finality-aleph/src/new_network/mod.rs

```diff
@@ -120,3 +126,15 @@ pub enum ConnectionCommand {
     AddReserved(HashSet<Multiaddr>),
     DelReserved(HashSet<PeerId>),
 }
+
+/// What can go wrong when sending data.
```

Comment:
Better comment? But you don't have to, since I really like this one :joy:

Reply:
I have no idea, I thought this was just a normal comment. <_<"

Reply:
Rephrased to possibly make it more "programmer" sounding, although I think this version was more natural. :crying_cat_face: